### DISCUSSION TITLE
How to add istanbul

### DISCUSSION THREAD

**Discussion by johnrazeur** [storybookjs/storybook#13904] [Help]
I'm trying to add istanbul to storybook, but I've got the following error message on each of my stories:

`Component.stories.tsx: don't know how to turn this value into a node`

I have installed babel-plugin-istanbul

`yarn add babel-plugin-istanbul --dev`

And I added this to my .storybook/main.js file:
```javascript
  babel: async (options) => ({
    ...options,
    plugins: [
      ...options.plugins,
      require.resolve('babel-plugin-istanbul'),
      // "istanbul" -> doesn't work too 
     ]
    // any extra options you want to set
  }),
```

If istanbul can't be add to storybook, what are my options to add instruments to my storybook project ?

**Comment by bichotll** [comment]
I also need this. Besides combining the report I'm getting from this and the one I'm getting from the normal jest tests.

Otherwise, it's not possible for us to know if we are missing an important part.
